src/components/common/HeaderMobile.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div
+        class="flex justify-between items-center lg:hidden px-4 py-1 sm:py-2 bg-white dark:bg-darkGray-800 shadow"
+    >
+        <button
+            class="-ml-3 p-2 rounded-full hover:bg-blue-100 dark:hover:bg-darkGray-600 hover:text-gray-900 focus:outline-none focus:ring focus:ring-blue-100 focus:bg-blue-50 group"
```

Comment:
I don't think it's intuitive to create a separate header component for mobile devices. A single header component should be responsive.